app/services/ApprovalService.scala

```diff
@@ -53,7 +56,21 @@ class ApprovalService @Inject() (repository: ApprovalRepository, reviewRepositor
         saveProcess(createApprovalProcess(process.meta.id, process.meta.title, initialStatus, jsonProcess)) flatMap {
           case Right(savedId) =>
             repository.getById(savedId) flatMap {
-              case Right(approvalProcess) => saveReview(createApprovalProcessReview(process, reviewType, approvalProcess.version))
+              case Right(approvalProcess) =>
+                pageBuilder.pages(process) match {
+                  case Right(pages) =>
+                    saveReview(ApprovalProcessReview(
+                                UUID.randomUUID(),
+                                process.meta.id, 
+                                approvalProcess.version,
+                                reviewType,
+                                process.meta.title,
+                                pageBuilder.fromPageDetails(pages)(ApprovalProcessPageReview(_,_,_))
+                              ))
+                  case _ =>
+                    Logger.error("Could not parse pages")
```

Comment:
I would provide details of process: id and version.

Reply:
oops

Reply:
will update